pass-bakery/app/controllers/APIController.scala

```diff
@@ -0,0 +1,38 @@
+package controllers
+
+import javax.inject._
+import play.api._
+import play.api.mvc._
+import play.api.libs.json._
+import io.circe.generic.extras.auto._
+import io.circe.syntax._
+import io.circe.parser
+import io.circe.generic.extras.Configuration
+import io.circe
+
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+@Singleton
+class APIController @Inject()(val controllerComponents: ControllerComponents) extends BaseController {
+
+  /**
+   * Create an Action to render an HTML page.
+   *
+   * The configuration in the `routes` file means that this method
+   * will be called when the application receives a `GET` request with
+   * a path of `/`.
+   */
+    //Define circe response, need to move out of controller when finished
+  case class Status(service: "pass-bakery", environment: String, serverTime: String)
```

Comment:
Generally model definitions are extracted into their own files, to be able to keep the logic of controllers clean. Please create a file in `app/models/` and declare the case class there

Additionally, anything related to a case class definition is better organized in a [companion object](https://docs.scala-lang.org/overviews/scala-book/companion-objects.html). You'll find that pattern used in every scala repository you see here, so lets change this to match it